cli/src/command/account.rs

```diff
@@ -680,20 +688,45 @@ pub async fn sync_command(account_handle: &AccountHandle) -> Result<(), Error> {
 }
 
 /// `transactions` command
-pub async fn transactions_command(account_handle: &AccountHandle) -> Result<(), Error> {
-    let transactions = account_handle.transactions().await?;
+pub async fn transactions_command(account_handle: &AccountHandle, show_details: bool) -> Result<(), Error> {
+    let mut transactions = account_handle.transactions().await?;
+    transactions.sort_by(|a, b| a.timestamp.cmp(&b.timestamp));
 
     if transactions.is_empty() {
         println_log_info!("No transactions found");
     } else {
-        for tx in transactions {
-            println_log_info!("{}", serde_json::to_string(&TransactionDto::from(&tx))?);
+        for (i, tx) in transactions.into_iter().enumerate() {
+            if show_details {
+                println_log_info!("{:#?}", tx);
+            } else {
+                let transaction_time = to_utc_date_time(tx.timestamp)?;
+                let formatted_time = transaction_time.format("%Y-%m-%d %H:%M:%S").to_string();
+
+                println_log_info!("{:<5} {:40}\t{:20}", i, tx.transaction_id.to_string(), formatted_time);
```

Comment:
What is `{:40}\t{:20}` for?
Don't see any difference with or without it
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=d1f4749c0a1bf0b9afe471e5bbf8c8e5

Reply:
this is a good point. `:40` is a minimum characters, but doesn't make sense here as id and date have always the same amount of characters